ro_help/hub/admin.py

```
@@ -121,6 +125,26 @@ def save_model(self, request, ngo, form, change):
         else:
             NGONeed.objects.filter(ngo=ngo, kind=KIND.MONEY).delete()
 
+    def _email_ngo_owners_about_creation(self, request, ngo, form, change):
+        if not change:
```

nit: Failing fast will increase readability 
```python
if change
    return
```

---

Good catch, thanks! Fixed.